examples/tv-casting-app/linux/main.cpp

```diff
@@ -158,31 +179,25 @@ void PrepareForCommissioning(const Dnssd::DiscoveredNodeData * selectedCommissio
     if (selectedCommissioner != nullptr)
     {
         // Send User Directed commissioning request
-        ReturnOnFailure(Server::GetInstance().SendUserDirectedCommissioningRequest(chip::Transport::PeerAddress::UDP(
-            selectedCommissioner->ipAddress[0], selectedCommissioner->port, selectedCommissioner->interfaceId)));
+        // Wait 1 second to allow our commissionee DNS records to publish (needed on Mac)
```

Comment:
Sleep is generally discouraged as a method of synchronization.

Reply:
I will address this in https://github.com/project-chip/connectedhomeip/issues/17299